src/client/transport/local.ts

```diff
@@ -161,8 +161,13 @@ export class LocalTransport extends Transport {
     this.isConnected = true;
   }
 
+  /**
+   * Called when any player sends a chat message and the
+   * master broadcasts the update to other clients (including
+   * this one).
+   */
   onChatMessage(matchID, chatMessage) {
-    this.master.onChatMessage(matchID, chatMessage);
+    this.master.onChatMessage(matchID, chatMessage, this.credentials);
```

Comment:
Same comment again here — would be good to type this just like the other transport methods.